benchmarks/benchmark.h

```diff
@@ -61,6 +69,9 @@ struct benchmark_arguments {
         validProgramOptions(true) {
     /* Match parameters */
     std::regex output_regex("--output");
+    std::regex device_regex("--device");
+    // device_vendor = std::string(".*");
+    // device_type   = std::string("*");
```

Comment:
Ah, that was some changes I made while working out how to handle wildcards, and where the `--device` specifier isn't given. I'll remove them.